docs/source/conf.py

```diff
@@ -102,26 +102,28 @@
     "light_logo": "manim-logo-sidebar.svg",
     "dark_logo": "manim-logo-sidebar-dark.svg",
     "light_css_variables": {
-        "color-content-foreground": "#413c3c",
-        "color-background-primary": "#f1ece9",
-        "color-background-border": "#ded6d4",
-        "color-sidebar-background": "#ece6e2",
-        "color-brand-content": "#2196f3",
-        "color-brand-primary": "#525893",
-        "color-link": "#e07a5f",
-        "color-link--hover": "#a05744",
-        "color-inline-code-background": "#f9f9f9",
+        "color-content-foreground": "#000000",
+        "color-background-primary": "#ffffff",
+        "color-background-border": "#ffffff",
+        "color-sidebar-background": "#f2f3f5",
```

Comment:
```suggestion
        "color-sidebar-background": "#ece6e2",
```

Reply:
this is the community background color

Reply:
I checked out the docs with this colour and I was not a fan of the look. I think it could afford to be a _touch_ lighter. But overall I don't see too many issues with it. Maybe I could introduce some variation with the link colour? A change from red to something else